server/routes/comment.js

```diff
@@ -0,0 +1,72 @@
+const express = require('express');
+const router = express.Router();
+// const { User } = require("../models/User");
+// const { Club } = require("../models/Club");
+const { Comment } = require("../models/Comment");
+
+function getKoreanDate(){
+    var date = new Date();
+    var year = date.getFullYear();
+    var month = date.getMonth();
+    var today = date.getDate();
+    var hours = date.getHours();
+    var minutes = date.getMinutes();
+    var seconds = date.getSeconds();
+    var milliseconds = date.getMilliseconds();
+    const currentDate = new Date(Date.UTC(year, month, today, hours, minutes, seconds, milliseconds));
+    return currentDate.toISOString().slice(0,10);
+}
+
+router.post('/saveComment',(req,res)=>{
+    const variables = {
+        ...req.body,
```

Comment:
```suggestion
        ...req.body,
```
좋은 코드인 것 같아요 저도 참고하겠음 😎

Reply:
🤩